maquetado/scss/tech.scss

```diff
@@ -0,0 +1,50 @@
+%equal-align {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+}
+
+.tech {
+  @extend %equal-align;
+
+  background-color: $green;
+  border-radius: 5px;
+  max-width: 200px;
+  padding: 1rem;
+  position: relative;
+
+  &::before {
+    border: 1px solid $green;
+    border-radius: 5px;
+    content: '';
+    height: 100%;
+    left: -10px;
+    position: absolute;
+    top: -10px;
+    width: 100%;
+  }
+
+  &-logo {
```

Comment:
I really liked this ampersand touching! great work 👍

Reply:
Wow! I didn't know this, good work!